org-eclipse-che-jdt-ls-extension/jdt.ls.extension.api/src/main/java/org/eclipse/che/jdt/ls/extension/api/dto/OrganizeImports.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.jdt.ls.extension.api.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** @author Anatolii Bazko */
+public class OrganizeImports {
+  private String resourceUri;
+  private List<String> choices;
+
+  public OrganizeImports() {
+    this.choices = new ArrayList<>();
+  }
+
+  public OrganizeImports(String resourceUri) {
+    this.resourceUri = resourceUri;
+    this.choices = new ArrayList<>();
+  }
+
+  public OrganizeImports(String resourceUri, List<String> choices) {
```

Comment:
Please document what the values for "resourceUri" and "choices" can be. This is API, so documentation should be proper.
As I understand it, you have to put in fully qualified type names?